apps/server/src/shared/infra/identity-management/keycloak/service/keycloak-seed.service.spec.ts

```diff
@@ -182,6 +183,8 @@ describe('KeycloakSeedService', () => {
 		kcApiUsersMock.create.mockRestore();
 		kcApiUsersMock.del.mockRestore();
 		kcApiUsersMock.find.mockRestore();
+		jest.clearAllMocks();
+		jest.resetModules();
```

Comment:
what make resetModules() ?

Reply:
we mock jest.mock('node:fs/promises') here and resetModules() resets this, JEST issus "memory leak".